options/options.go

```diff
@@ -235,8 +235,8 @@ func (terragruntOptions *TerragruntOptions) InsertTerraformCliArgs(argsToInsert
 	// command is either 1 word or 2 words
 	var args []string
 	args = append(args, terragruntOptions.TerraformCliArgs[:commandLength]...)
-	args = append(args, argsToInsert...)
 	args = append(args, terragruntOptions.TerraformCliArgs[commandLength:]...)
+	args = append(args, argsToInsert...)
```

Comment:
Hm, doesn't this bypass all the `commandLength` logic above and just insert the args at the end? Does that work with all Terraform commands? That is, do `plan`, `apply`, `destroy`, `state mv`, `force-unlock <ID>`, etc all expect the args at the very end? Or do some need it in the middle?

Reply:
I think you are right this might not work for all commands, I think the solution I'm proposing here it's a bit sloppy and just for my specific just case